examples/flutter_gallery/lib/demo/progress_indicator_demo.dart

```diff
@@ -73,6 +74,7 @@ class _ProgressIndicatorDemoState extends State<ProgressIndicatorDemo> with Sing
       new Row(
         mainAxisAlignment: MainAxisAlignment.spaceEvenly,
         children: <Widget>[
+          new CupertinoActivityIndicator(),
```

Comment:
I think we probably need a long term solution for cupertino and gallery though for the short term, I think it's good to have an easy comparison between different widget skins of the same widget 'type'.

Might be confusing for the user if we sliced both horizontally with widget 'function categories' and vertically with 'widget look/use circumstance'.